scripts/process_uploads.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+"""
+Processes a new batch of samples from the UPLOAD bucket,
+into the MAIN and ARCHIVE buckets using the batch_move_files function. 
+
+Determines the files to be moved through a comparison of the 
+most recently updated CSV and the CSV uploaded in a previous batch run. 
+"""
+
+import os
+import io
+import csv
+from os.path import join, basename
+import subprocess
+from typing import List
+import click
+import hail as hl
+import hailtop.batch as hb
+from google.cloud import storage
+from joint_calling.upload_processor import batch_move_files
+
+
+def samples_from_csv(bucket_name, path):
+    """ Determines list of samples from a given csv file. """
+
+    client = storage.Client()
+    bucket = client.get_bucket(bucket_name)
+    full_path = os.path.join('gs://', path, '*.csv')
+
+    cmd = f'gsutil ls \'{full_path}\''
+    csv_path = subprocess.check_output(cmd, shell=True).decode().strip()
+    prefix = len('gs:///') + len(bucket_name)
+    blob = bucket.get_blob(csv_path[prefix:])
+    data = blob.download_as_string().decode('utf-8')
```

Comment:
`download_as_string` is deprecated.

```suggestion
    data = blob.download_as_bytes().decode('utf-8')
```

Reply:
Actually `download_as_text` might be even better, as you don't need to call `decode`.

Reply:
Thanks! Have fixed.